vanilla-framework/vanilla-brochure-theme
# Brochure docs improvements

## Done

Fixed naming, spelling and title case errors

## QA

- Pull code
- Run `./run serve --watch`
- Check code for spelling title case changes
- Check that http://0.0.0.0:8102/vanilla-brochure-theme/examples/patterns/lists/list-with-icon/ works

## Details

Related https://github.com/ubuntudesign/docs.vanillaframework.io/issues/100
